README.md

```diff
@@ -48,6 +48,7 @@ const doctor = new ClinicDoctor()
   * sampleInterval [`<number>`][] Time between each sample in milliseconds.
     **Default**: 10
   * detectPort [`<boolean>`][] **Default**: false
+  * dataPath [`<string>`][] The folder where the collected data is stored. **Default**: '.'
```

Comment:
Suggestion: rename the option to `path` as there are no other `path` folders I can think off. Internally it's also called `path`.